### PR TITLE
Tcvp 155

### PR DESCRIPTION
**Pull request by hpatel1994**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: infrastructure/README.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
 # Infrastructure
+Generating kubernetes yaml file using docker-compose.yaml to launch and deploy the project inside the OC4
+- Using Kompose kubernetes tool
+ - Install Kompose and add your directory location into your environment  settings PATH variable.
+ - Inside the terminal go to your docker-compose.yaml location of the project
+ - Run command "kompose convert --provider=openshift --build=build-config --build-repo=https://github.com/bcgov/jag-traffic-courts-online.git" this will convert all the service inside the docker-compose.yaml to kubernetes yaml format and will generate build configs
+ - After that Run command "kompose convert --provider=openshift --build=deploy-config --build-repo=https://github.com/bcgov/jag-traffic-courts-online.git" this will convert all the service inside the docker-compose.yaml to kubernetes yaml format and will generate deployment configs
+
```
Comment (rantifamilusi) [inline infrastructure/README.md:8]: - Can we include the link to Kompose tool and how to install it
- Where will the yaml files be generated ? Can we include the path to look out for ?
-

File: infrastructure/README.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+1.) Deploying Dispute-api in the OC4 Environment
+    - dispute-api-deploymentconfig.yaml
+     This is used to generate the deployment configs for the dispute api inside the OC4 environment.
+     Copy the content of insfrastructure/openshift/dispute-api-deploymentconfig.yaml into deployment config of the dev namespace of traffic courts project in OC4
+    - dispute-api-imagestream.yaml
+     This file is used to generate and save the Dotnet core image inside the OC4 repository
+     Copy the content of insfrastructure/openshift/dispute-api-imagestream.yaml   yaml into the imagestream of the tools namespace of the traffics courts online in OC4
+    -dispute-api-service.yaml
+     This file is to generate the service which can define the ports for the backend api
+     Copy the content of insfrastructure/openshift/dispute-api-service.yaml into the service of dev namespace of the traffic courts online projects in OC4
```
Comment (rantifamilusi) [inline infrastructure/README.md:19]: is there a cli option for this ? like oc login & oc process 

(look at this read me --> https://github.com/bcgov/fams3/tree/master/openshift)

If it's possible lets add that as an option.

**Review by rantifamilusi — COMMENTED** [review]
Looks good. Few comments

**Review by poggdogg — APPROVED** [review]
Looks good

**Review by rantifamilusi — APPROVED** [review]

**Pull request merged**